forge-gui/res/cardsfolder/upcoming/twice_upon_a_time_unlikely_meeting.txt

```diff
@@ -0,0 +1,16 @@
+Name:Twice Upon a Time
+ManaCost:4 U U
+Types:Sorcery
+A:SP$ AddTurn | NumTurns$ 1 | SubAbility$ DBChange | IsPresent$ Doctor.YouCtrl | PresentCompare$ GE2 | SpellDescription$ Cast this spell only if you control two or more Doctors.,,,,,,Take an extra turn after this one.

```

Comment:
@Hanmac 
do you think it's cleaner if it gets written as CantBeCast LT2 Doctor?

Reply:
but maybe it would mess up the adventure part 🤔

Reply:
i mean it can work as CantBeCast, but i think a Restriction like this should be enough